src/session/session_api.c

```diff
@@ -166,8 +166,7 @@ __session_alter(WT_SESSION *wt_session, const char *uri, const char *config)
 	WT_WITH_CHECKPOINT_LOCK(session,
 	    WT_WITH_SCHEMA_LOCK(session,
 		ret = __wt_schema_worker(session, uri, __wt_alter, NULL, cfg,
-		WT_BTREE_ALTER |
-		WT_DHANDLE_DISCARD_FORCE | WT_DHANDLE_EXCLUSIVE)));
```

Comment:
Removing the DISCARD_FORCE is fine here.  When doing this I added ALTER to the special flags much later in the timeline and did not remember/realize and go back to delete the other flag.  I've been running alter tests for hours and all is well.  So this LGTM.